src/dBFT/DBFTPlugin.cs

```diff
@@ -0,0 +1,34 @@
+using Akka.Actor;
+using Neo.Network.P2P;
+using Neo.Plugins;
+using Neo.Wallets;
+
+namespace Neo.Consensus
+{
+    public class DBFTPlugin : Plugin, IConsensusProvider, IP2PPlugin
+    {
+        private IActorRef consensus;
+
+        public DBFTPlugin()
+        {
+            AddService(this);
+        }
+
+        protected override void Configure()
+        {
+            Settings.Load(GetConfiguration());
+        }
+
+        void IConsensusProvider.Start(Wallet wallet)
+        {
+            consensus = System.ActorSystem.ActorOf(ConsensusService.Props(System.LocalNode, System.TaskManager, System.Blockchain, System.LoadStore(Settings.Default.RecoveryLogs), wallet));
+        }
+
+        bool IP2PPlugin.OnP2PMessage(Message message)
```

Comment:
Should we `void IP2PPlugin.OnVerifiedInventory(IInventory inventory)` from core?

Reply:
https://github.com/neo-project/neo/pull/2216

Reply:
Here we need unverified transactions.